app/src/main/java/com/github/leuludyha/ibdb/presentation/components/auth/signup/ChangeUsernamePrompt.kt

```diff
@@ -49,7 +49,7 @@ object ChangeUsernamePrompt : SignUpPromptBase(
             text =
             stringResource(id = R.string.prompt_username_title_begin) +
                     authContext.principal.username +
-                    stringResource(id = R.string.prompt_username_title_end)
+                    stringResource(id = R.string.prompt_username_title_end),
         )
```

Comment:
Nitpick: Try to avoid these kind of changes and files when commiting, my guess is that it is a leftover from some test you were doing, but doesn't add anything besides that weird comma